### ISSUE TITLE
Amidst is not able to find your .minecraft directory

### ISSUE THREAD

**Issue by RacoonSauce** [toolbox4minecraft/amidst#931]
Literally been looking through everything for hours, I have no coding experience and just want to use amidst with MultiMC, I have seen other threads saying to use the -mcpath parameter but I have no idea how to use it and what a command line interface is, I have tried it on powershell and I get the error. And when I put java -jar amidst-v4-6.jar -mcpath C:\Program Files\MultiMC\instances\1.16.4Fabric\.minecraft into powershell, it says argument not allowed, also no argument allowed on cmd. What can I do? I know nothing about programming so you might have to dumb it down

**Comment by burgerguy** [comment]
if you didn't fix this, you probably just need to put quotes around the stuff after mcpath, like this:
`java -jar amidst-v4-6.jar -mcpath "C:\Program Files\MultiMC\instances\1.16.4Fabric.minecraft"`
